src/main/resources/public/raml/resourceTypes/status.raml

```diff
@@ -0,0 +1,8 @@
+description: This class returns status information about Eiffel Intelligence and it's dependencies.
```

Comment:
Perhaps use another word instead of dependencies. Some may get confused since this terminology is used for maven libraries.

Reply:
I've seen you use the phrase 'dependent services', I think that would sound better here.

Reply:
Done.